README.md

```diff
@@ -229,26 +229,26 @@ As work in progress, there are plently of issues and features missing, which we
 The Conclave Core SDK consists of several technologies working together. Here is a description of the most
 meaningful directories in this repo.
```

Comment:
The word _meaningful_ doesn't seem to fit here. Please rewrite as: `Here is a description of the most _important_ directories in this repo`.

Reply:
Done.